### DISCUSSION TITLE
How to restrict access to rendered config on a device?

### DISCUSSION THREAD

**Discussion by andershagman** [netbox-community/netbox#17206] [Q&A]
I use view permissions on devices and interfaces on our switches to everyone in IT. This because I use interface descriptions to document endpoints and I don´t have to answer all those questions.

Now I have started to use render config and to my absolute horror I saw that everyone can read the config of the switches.
I can restrict viewing power ports and changelog but not the rendered config.
Or can I, have I missed somthing?

**Comment by ypid** [comment]
Looks like https://github.com/netbox-community/netbox/issues/16681 should solve this.
